how-to/software/kernel-crash-dump.md

```diff
@@ -83,16 +66,9 @@ Select 'Yes' to select `kexec-tools` for all reboots.
  |---------------------------------------------------------------------------|
 ```
 
-'Yes' should be selected here as well, to enable `kdump-tools`.
-
-If you ever need to manually enable the functionality, you can use the  `dpkg-reconfigure kexec-tools` and `dpkg-reconfigure kdump-tools` commands and answer 'Yes' to the questions. You can also edit `/etc/default/kexec` and set parameters directly:
-
-```text
-# Load a kexec kernel (true/false)
-LOAD_KEXEC=true
-```
+'Yes' should be selected to enable `kdump-tools`.
 
-As well, edit `/etc/default/kdump-tools` to enable `kdump` by including the following line:
+If you ever need to manually enable the functionality, you can use the `dpkg-reconfigure kdump-tools` command and answer 'Yes'. You can also edit `/etc/default/kdump-tools` to enable `kdump` by including the following line:
```

Comment:
This is only talking about enabling, what if I change my mind and want it disabled? Therefore I think the text above would be better if it basically said that if you want to revisit your choice, you can run `sudo dpkg-reconfigure kdump-tools`, or edit the default file you mention. Assuming both reach the same outcome (I wonder if the debconf db will be kept in sync with the default file... what happens if you set "no" in /etc/default and run dpkg-reconfigure, will that also be "no"?)

Reply:
@bryceharrington ? :)

Reply:
Thanks for the review, sorry it fell off my radar!

I've updated the text to reflect enabling AND disabling, and clarified that the editing of /etc/default/kdump-tools is required regardless.